src/libasr/pass/pass_manager.h

```diff
@@ -129,7 +129,6 @@ namespace LCompilers {
                 "select_case",
                 "inline_function_calls",
                 "unused_functions",
-                "pass_compare"
```

Comment:
Probably this means we need backend specific list of passes. Because C/C++/Julia will need this pass but LLVM won't.